lib/ansible/executor/task_queue_manager.py

```diff
@@ -167,6 +167,10 @@ def load_callbacks(self):
                                    " see the 2.4 porting guide for details." % callback_obj._load_name, version="2.9")
             self._callback_plugins.append(callback_obj)
 
+        for callback_plugin_name in (c for c in C.DEFAULT_CALLBACK_WHITELIST if '.' in c):
```

Comment:
a . might not be best identifier of collection based callback (2 at least)

Reply:
fixed to use "is this a valid collection identifier" func